fdm-agents/src/runners/stream.test.ts

```
@@ -0,0 +1,192 @@
+        }
+        const events = await collect(runStreamAgent(agent as any, "test"))
+        expect(events).toHaveLength(1)
+        expect(events[0].event).toBe("error")
+        expect(events[0].data.message).toBe("Agent crashed")
+    })
+
+    it("should yield an error event with 'Unknown error' for non-Error throws", async () => {
+        const agent = {
+            stream: () => {},
+            streamEvents: () => {
+                // biome-ignore lint/style/noThrowLiteral: intentional for test coverage
```

Biome (2.5.0)

[error] Failed to parse category "lint/style/noThrowLiteral". Example of suppression: // biome-ignore lint: reason

(suppressions/parse)